assets/uswds-sass/core/_base.scss

```diff
@@ -0,0 +1,58 @@
+// Apply a natural box layout model to all elements, but allowing components to
+// change
+
+@if $theme-global-border-box-sizing {
+  html {
+    box-sizing: border-box;
+  }
+
+  *,
+  *::before,
+  *::after {
+    box-sizing: inherit;
+  }
+}
+
+body {
+  background-color: color('white');
+  color: color('ink');
+  overflow-x: hidden;
+}
+
+// Hack for clearfixes
+.lt-ie9 {
+  * {
+    filter: none !important; /* stylelint-disable-line declaration-no-important */
+  }
+}
+
+// for IE < 11, see:
```

Comment:
These files are brought into the project with gulp, so there's might not much point to editing them because they'll just return if we update USWDS. Had to bring all of these files in so I could use the USWDS variables and functions in the USDS project Sass files.

Reply:
I agree, I'll submit a PR to USWDS when I get a chance so these are fixed on their side.